scripts/provision-member-directories.sh

```diff
@@ -155,7 +156,8 @@ EOL
 **/backend.tf @ministryofjustice/modernisation-platform
 **/subnet_share.tf @ministryofjustice/modernisation-platform
 **/networking.auto.tfvars.json @ministryofjustice/modernisation-platform
-/terraform/modules
+**/platform_*.tf @ministryofjustice/modernisation-platform
+/terraform/modules @ministryofjustice/modernisation-platform
```

Comment:
The /terraform/modules folder is for users to create their own modules without needing approval from us, so that can be left as it is with no codeowner

Reply:
updated